package.json

```diff
@@ -53,15 +54,17 @@
     "build:examples": "rimraf ./build/examples/* && node tasks/build-examples.js && webpack --config webpack.examples.config.js",
     "build:docs": "jsdoc --package ./package.json --readme ./README.md -c .jsdoc",
     "build:js": "webpack --config webpack.development.config.js && webpack -p --config webpack.production.config.js",
-    "build": "npm run test && npm run build:examples && npm run build:docs && npm run build:js"
+    "build": "npm run clean && BABEL_ENV=build babel src -d dist"
   },
   "dependencies": {
     "antd": "2.12.8",
     "lodash": "4.17.4",
     "loglevel": "1.4.1",
     "ol": "4.3.2",
+    "prop-types": "15.5.10",
```

Comment:
This migfht be one of the causes of the merge conflict

Reply:
I don't get why adding a line causes conflicts but you were right.